Module/src/PSWordCloud/NewWordCloudCommand.cs

```diff
@@ -93,7 +93,7 @@ public string Path
         {
             get => _resolvedPath;
             set => _resolvedPath = SessionState.Path
-                .GetUnresolvedProviderPathFromPSPath(value, out ProviderInfo provider, out PSDriveInfo drive);
+                .GetUnresolvedProviderPathFromPSPath(value, out _, out _);
```

Comment:
You only support file system right? I usually throw early if the resolved provider is something else.  You can check with

```csharp
provider.Name.Equals(FileSystemProvider.ProviderName, StringComparision.Ordinal)
```

Reply:
Hmm. In _theory_ we **can** support whatever. SVG files are just giant XML blobs essentially, so...

Hmm. Maybe.

Reply:
Do you use provider API's to write the content though? If you do then yeah for sure definitely don't throw 🙂

Reply:
🤔 That's a question. Not actually sure. Might do. Might be a future improvement. 😉

Reply:
Aaand as you know I've gone ahead and added support for whichever psprovider the user wants to work with lol